src/coreclr/nativeaot/Runtime/windows/PalRedhawkMinWin.cpp

```diff
@@ -503,10 +503,17 @@ REDHAWK_PALEXPORT CONTEXT* PalAllocateCompleteOSContext(_Out_ uint8_t** contextB
     }
 #endif //TARGET_X86
 
-    // Determine if the processor supports AVX or AVX512 so we could
-    // retrieve extended registers
+#if defined(TARGET_X86) || defined(TARGET_AMD64)
+    const DWORD64 xStateFeatureMask = XSTATE_MASK_AVX | XSTATE_MASK_AVX512;
+    const ULONG64 xStateCompactionMask = XSTATE_MASK_LEGACY | XSTATE_MASK_MPX | xStateFeatureMask;
+#elif defined(TARGET_ARM64)
+    const DWORD64 xStateFeatureMask = XSTATE_MASK_ARM64_SVE;
+    const ULONG64 xStateCompactionMask = XSTATE_MASK_LEGACY | xStateFeatureMask;
+#endif
+
+    // Determine if the processor supports extended features so we could retrieve those registers
     DWORD64 FeatureMask = GetEnabledXStateFeatures();
```

Comment:
This isn't defined for ARM64 until v10.0.26100.0; going to need to dynamically resolve it instead like we do for `pfnInitializeContext2`

Not going to do that tonight, just leaving a note for myself for tomorrow

Reply:
Do you refer to Windows SDK version or a Windows version?

Reply:
Both, it's a new API for Arm64 in v10.0.261000.

Hence we try to resolve the export and don't do anything on Arm64 if it can't be resolved.